ext/session/session.stub.php

```diff
@@ -85,7 +85,7 @@ function session_cache_limiter(?string $value = null): string|false {}
 
 function session_cache_expire(?int $value = null): int|false {}
 
-function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain = null, ?bool $secure = null, ?bool $httponly = null): bool {}
+function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain = null, ?bool $secure = null, ?bool $httponly = null, SameSite $sameSite = SameSite::Lax): bool {}
```

Comment:
just general feeling impression, the camel case feels out of place whereas is mostly lower case with underscore otherwise.

Reply:
ACK